shared/constants/entities.js

```diff
@@ -24,12 +25,14 @@ export type Actions = Delete | Merge | Replace
 // State
 export type State = KBRecord<{
   devices: Map<string, DeviceDetailRecord>,
+  teams: Map<Teams.Teamname, Teams.TeamRecord>,
```

Comment:
This is wrong, it should be:
`teams: Team.TeamRecord`

Reply:
???